pkg/operator/sync_test.go

```diff
@@ -311,17 +355,17 @@ func TestSyncMutatingWebhooks(t *testing.T) {
 	optr.syncHandler = nil
 
 	configuration, _ := runtime.DefaultUnstructuredConverter.ToUnstructured(defaultConfiguration.DeepCopy())
-	testSyncWebhookConfiguration(t,
-		optr.dynamicClient.Resource(admissionregistrationv1.SchemeGroupVersion.WithResource("mutatingwebhookconfigurations")),
+	optr.testSyncWebhookConfiguration(t,
+		admissionregistrationv1.SchemeGroupVersion.WithResource("mutatingwebhookconfigurations"),
 		&unstructured.Unstructured{Object: configuration},
 		optr.syncMutatingWebhook,
 		stop,
 		optr.mutatingWebhookListerSynced, testCases)
 }
 
-func testSyncWebhookConfiguration(
+func (optr *Operator) testSyncWebhookConfiguration(
```

Comment:
Why did this change to being a member of the `Operator`? Shouldn't the test code be completely separate from the `Operator`?

Reply:
Did this change authomatically, fixed this to be an argument.